docs/api/ReactWrapper/renderProp.md

```diff
@@ -1,4 +1,4 @@
-# `.renderProp(propName, ...args) => ReactWrapper`
+# `.renderProp(propName)(...args) => ReactWrapper`
 
 Calls the current wrapper's property with name `propName` and the `args` provided.
```

Comment:
Let's reword this; something like "Returns a function that, when called with arguments `args`, will return a new wrapper based on the render prop in the original wrapper's prop `propName`."